sql/core/src/test/scala/org/apache/spark/sql/execution/GlobalTempViewSuite.scala

```diff
@@ -123,6 +123,19 @@ class GlobalTempViewSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("CREATE TABLE LIKE USING provider for view") {
+    withTable("cloned") {
+      withGlobalTempView("src") {
+        sql("CREATE GLOBAL TEMP VIEW src AS SELECT 1 AS a, '2' AS b")
+        sql(s"CREATE TABLE cloned LIKE $globalTempDB.src USING orc")
```

Comment:
This is nothing special with global temp view, we can just put the test in `DDLSuite`

Reply:
Ok, I will remove it